src/lib.rs

```diff
@@ -5,14 +5,24 @@
 // along with the Commit library. If not, see <https://mit-license.org/>.
 
 use arbitrary::{Arbitrary, Unstructured};
+#[cfg(feature = "ark-serialize")]
```

Comment:
nitpick: I like grouping all the `#[cfg(feature = x)]` imports together, it's a little bit neater